sycl/test-e2e/Matrix/SG32/element_wise_all_ops.cpp

```diff
@@ -10,6 +10,9 @@
 // SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
 // UNSUPPORTED: gpu-intel-dg2
 
+// TODO: Reenable, see https://github.com/intel/llvm/issues/14598
+// UNSUPPORTED: windows, linux
```

Comment:
The issue in question has been fixed. No need to add "unsupported"

Reply:
Done